src/axom/sidre/core/Group.hpp

```diff
@@ -1152,16 +1156,18 @@ class Group
   /*!
    * \brief Create a child Group within this Group with given name or path.
    *
-   * If name is an empty string or Group already has a child Group with
-   * given name or path, method is a no-op.
+   * If name is an empty string, method is a no-op.
```

Comment:
```suggestion
   * If name or path is an empty string, method is a no-op.
```

For consistency with other statements in this comment section.